just_audio/android/src/main/java/com/ryanheise/just_audio/AudioPlayer.java

```diff
@@ -660,10 +678,31 @@ public void setVolume(final float volume) {
     }
 
     public void setSpeed(final float speed) {
-        player.setPlaybackParameters(new PlaybackParameters(speed));
+        _speed = speed;
+        if (_skipSilence) {
+            player.setPlaybackParameters(new PlaybackParameters(_speed, 1, _skipSilence));
+        } else {
+            player.setPlaybackParameters(new PlaybackParameters(speed));
+        }
         broadcastPlaybackEvent();
     }
 
+    public void setSkipSilence(final boolean skipSilence) {
+        _skipSilence = skipSilence;
+        player.setPlaybackParameters(new PlaybackParameters(_speed, 1, skipSilence));
+    }
+
+    public void setVolumeBoost(final boolean enabled, final int gainmB) {
+        if (android.os.Build.VERSION.SDK_INT >= 19 && audioSessionId != null) {
+            _volumeBoostEnabled = enabled;
+            _volumeBoostGainMB = gainmB;
+            Log.e(TAG, "setVolumeBoost in android 2 : " + enabled);
+            loudness = new LoudnessEnhancer(audioSessionId);
+            loudness.setEnabled(enabled);
+            loudness.setTargetGain(gainmB);
```

Comment:
Question: how many AudioEffects do we want to support? It may be worth designing the API considering this bigger picture, and also iOS.